### PR TITLE
Andrew Huffman - PR changes

### PR DESCRIPTION
**Pull request by ahuffman**
Some changes I made a few months back, as we discussed today.

### CODE REVIEW

**Comment by wtcross** [comment]
Awesome, thank you for submitting this! I noticed a few things that should be addressed. I'll submit a comment for each of them.

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) 2016 Tyler Cross
+Copyright (c) 2016 Andrew Huffman
```
Comment (wtcross) [inline LICENSE:4]: 👍

File: README.md [hunk 2]
```diff
@@ -1,68 +1,129 @@
-# sudoers
-> An Ansible role for comprehensive management of a sudoers configuration.
-
-[![Build Status](https://travis-ci.org/wtcross/ansible-sudoers.svg?branch=master)](https://travis-ci.org/wtcross/ansible-sudoers)
+# ahuffman.sudoers
```
Comment (wtcross) [inline README.md]: I'd like to give you contributor access to this repo. That said, I think this'll still need to be changed to wtcross.sudoers.

File: README.md [hunk 3]
```diff
@@ -1,68 +1,129 @@
...
 This role makes it possible to completely define your sudoers configuration with Ansible. All of the following are configurable:
 - defaults
 - aliases
+  * Users
+  * Runas
+  * Hosts
+  * Commands
 - specifications
 
-*Tip:* Here's a [great document about sudoers configuration](https://help.ubuntu.com/community/Sudoers)
```
Comment (wtcross) [inline README.md:11]: Is there a reason you removed this note? I don't care about the source, but wanted the README in this role to point to some comprehensive sudoers guide. Open to suggestions on an alternative.

File: README.md [hunk 4]
```diff
@@ -113,6 +177,14 @@ defaults: string|[string]
     - `string`
     - `string: string`
     - `string: [string]`
+```
 
 ## License
 [MIT](LICENSE)
+
+## Author Information
+### Original Author:
```
Comment (wtcross) [inline README.md]: Rather than specifying original author, how about just listing each of us as an author?

File: meta/main.yml [hunk 5]
```diff
@@ -1,10 +1,11 @@
 galaxy_info:
-  author: Tyler Cross
-  description:
-  issue_tracker_url: https://github.com/wtcross/ansible-sudoers/issues
```
Comment (wtcross) [inline meta/main.yml:4]: If the code is going to live in this repo, can we just keep this url?

Comment (ahuffman) [reply meta/main.yml:4]: Fixed in latest commit

File: README.md (hunk 4 above)
Comment (ahuffman) [reply README.md]: Changed to include us both in latest commit.

File: README.md (hunk 3 above)
Comment (ahuffman) [reply README.md:11]: I was assuming users of this role would have experience manipulating /etc/sudoers, but I have no problem adding it back in.  A Fedora/CentOS resource might be better in the future ;).  Added back in latest commit.

File: README.md (hunk 2 above)
Comment (ahuffman) [reply README.md]: Changed back to wtcross.sudoers in latest commit.

File: tasks/main.yml [hunk 6]
```diff
@@ -4,20 +4,20 @@
     name: sudo
     update_cache: yes
     state: present
-  when: "{{ ansible_os_family == 'RedHat' }}"
+  when: ansible_os_family == 'RedHat'
```
Comment (wtcross) [inline tasks/main.yml]: In order to make this following best practices can you wrap it with template tags?

File: tasks/main.yml [hunk 7]
```diff
@@ -4,20 +4,20 @@
...
 
 - name: Ensure sudo is installed for Debian family OSs
   apt:
     name: sudo
     update_cache: yes
     state: present
-  when: "{{ ansible_os_family == 'Debian' }}"
+  when: ansible_os_family == 'Debian'
```
Comment (wtcross) [inline tasks/main.yml]: In order to make this following best practices can you wrap it with template tags?

File: tasks/main.yml [hunk 8]
```diff
@@ -4,20 +4,20 @@
...
 
 - name: Ensure sudo is installed for SUSE family OSs
   zypper:
     name: sudo
     state: present
-  when: "{{ ansible_os_family == 'SUSE' }}"
+  when: ansible_os_family == 'SUSE'
```
Comment (wtcross) [inline tasks/main.yml]: In order to make this following best practices can you wrap it with template tags?

File: tasks/main.yml (hunk 6 above)
Comment (wtcross) [reply tasks/main.yml]: Ex: `"{{ ansible_os_family == 'RedHat' }}"`

Comment (wtcross) [reply tasks/main.yml]: Otherwise Ansible will throw a warning when running the playbook.

Comment (wtcross) [reply tasks/main.yml]: See the porting guide and look for bare variables: https://docs.ansible.com/ansible/porting_guide_2.0.html

File: tasks/main.yml (hunk 8 above)
Comment (ahuffman) [reply tasks/main.yml]: Should be all set now

File: tasks/main.yml (hunk 7 above)
Comment (ahuffman) [reply tasks/main.yml]: Should be all set now

File: tasks/main.yml (hunk 6 above)
Comment (ahuffman) [reply tasks/main.yml]: Should be all set now

**Comment by wtcross** [comment]
Thanks for the quick responses! Merging now.

**Pull request merged**